### PR TITLE
chore(guides): guides/_subsections/automated-analysis.md screenshots updated

### PR DESCRIPTION
**Pull request by aali309**
See: #147 
Updated the screenshot in `guides/_subsections/automated-analysis.md`

### CODE REVIEW

**Comment by aali309** [comment]
Updated the screen shots.. I will reword it on the written PR that im woriking on

File: guides/_subsections/automated-analysis.md [hunk 1]
```diff
@@ -52,30 +52,6 @@ Cryostat also provides an *Automated Analysis Card* that is able to display the
...
-          <img src="{{ site.url }}/images/2.4.0/automated-analysis-5.png">
-        </a>
-        <figcaption>
-          To download the HTML <i>Automated Analysis</i> report to local disk, right click 
-          the page and select <i>Save Page As...</i>. Alternatively, press <kbd>Ctrl</kbd>+<kbd>S</kbd> 
-          on Windows/Linux, or <kbd>⌘</kbd>+<kbd>S</kbd> on macOS.
-        </figcaption>
-      </figure>
-    </p>
-    {% endcapture %}
```
Comment (andrewazores) [inline guides/_subsections/automated-analysis.md:78]: Could you undo this change and put it into the next PR for updating the written content? Just so that the final upstream PRs that will be merged into the release branch are as cleanly separated as possible and any rebase/merge conflicts are as easy to understand as possible.

Comment (andrewazores) [reply guides/_subsections/automated-analysis.md:78]: This didn't quite get undone - it looks like this has the copy from `main` instead of the copy from the base branch, or something like that.

**Review by andrewazores — COMMENTED** [review]
Allllmost done.

This should not be replacing the existing 2.3.0 image version, it should be updating the 2.4.0 copy. Otherwise what is going to happen is that the outdated image will be shown in the latest (2.4.0) docs, and the updated image will be shown if you select the 2.3.0 version on the website, which will not match.

**Comment by aali309** [comment]
I will update PR [#153](https://github.com/cryostatio/cryostatio.github.io/pull/153) and remove the screenshots that I added to the wrong folder (`2.3.0`) and just keep it to just edit written content.

File: guides/_subsections/automated-analysis.md [hunk 2]
```diff
@@ -65,8 +65,8 @@ Cryostat also provides an *Automated Analysis Card* that is able to display the
         </figcaption>
       </figure>
       <figure>
-        <a href="{{ site.url }}/images/2.4.0/automated-analysis-5.png" target="_blank">
-          <img src="{{ site.url }}/images/2.4.0/automated-analysis-5.png">
+        <a href="{{ site.url }}/images/2.3.0/automated-analysis-5.png" target="_blank">
+          <img src="{{ site.url }}/images/2.3.0/automated-analysis-5.png">
```
Comment (andrewazores) [inline guides/_subsections/automated-analysis.md:69]: Should these be referencing the new 2.4.0 images?

Comment (aali309) [reply guides/_subsections/automated-analysis.md:69]: No these are not needed in `2.4.0`  therefore should be removed( planned to do that on [#153](https://github.com/cryostatio/cryostatio.github.io/pull/153). I deleted the images in the `2.4.0` folder because we do not have the view report option just view in grafana.

Comment (aali309) [reply guides/_subsections/automated-analysis.md:69]: @andrewazores see the described changes above: [https://github.com/cryostatio/cryostatio.github.io/pull/153/commits/a8690f53e0c075b1219bf898187d624846121c79](https://github.com/cryostatio/cryostatio.github.io/pull/153/commits/a8690f53e0c075b1219bf898187d624846121c79)

Comment (aali309) [reply guides/_subsections/automated-analysis.md:69]: makes sense

**Review by andrewazores — APPROVED** [review]
